src/main/js/html.js

```diff
@@ -779,37 +777,38 @@ var backgroundColorAdjustSuffix = "BackgroundColorAdjust";
 
                 // Start element
                 if (context.ipd === "lr") {
-
```

Comment:
It's a bit of an idiomatic code style thing, but these blank lines are deliberate and endemic in the library, so we should probably keep them. I think I once got a pull request bounced back by @palemieux for breaking the coding style like this, and it looks like it would be good to contribute this workaround back to sandflow/imscjs if we can.